tests/framework/e2e/cluster.go

```diff
@@ -165,14 +165,14 @@ type EtcdProcessClusterConfig struct {
 
 	CipherSuites []string
 
-	ForceNewCluster     bool
-	InitialToken        string
-	QuotaBackendBytes   int64
-	NoStrictReconfig    bool
-	EnableV2            bool
-	InitialCorruptCheck bool
-	AuthTokenOpts       string
-	V2deprecation       string
+	ForceNewCluster            bool
+	InitialToken               string
+	QuotaBackendBytes          int64
+	NoStrictReconfig           bool
+	EnableV2                   bool
+	DisableInitialCorruptCheck bool
```

Comment:
I don't think this direction is much better. This is bad way to handle defaults, negating the configuration fields again makes it harder to track and understand.

Instead I would prefer that we separate struct definition from defaults. This way we can make the configuration struct more similar to "production" one and just have tests having different defaults.

Example of what I mean:
```
type  EtcdProcessClusterConfig struct {
...
  InitialCorruptionCheck bool
...
} 

func DefaultEtcdProcessClusterConfig() *EtcdProcessClusterConfig {
  return &EtcdProcessClusterConfig{
    InitialCorruptionCheck: true,
  }
}

func TestCorruption(t *testing.T) {
  config := DefaultEtcdProcessClusterConfig()
  config.InitialCorruptionCheck = false
  ...
}
```

Later we might decide to shorten shorten code by adding options for DefaultEtcdProcessClusterConfig, but this is separate issue.

Reply:
> I don't think this direction is much better. This is bad way to handle defaults, negating the configuration fields again makes it harder to track and understand.
> 
> Instead I would prefer that we separate struct definition from defaults. This way we can make the configuration struct more similar to "production" one and just have tests having different defaults.

Thanks. Actually it's a tradeoff. The good side is that we can leverage golang build-in feature of zero value (e.g. `false` for bool), so that the implementation can be simplified because we need to do nothing if we just want to use the default value; while the bad side it's counterintuitive (not disable == enable), and it isn't consistent with the product code just as you pointed out.

Let me follow your suggestion in this PR.  I will rollback the change to `InitialCorruptCheck`, and we can address it in a separate PR.

Reply:
Done.